android/app/google-services.json

```diff
@@ -0,0 +1,119 @@
+{
+  "project_info": {
+    "project_number": "854811651919",
+    "firebase_url": "https://status-react-app.firebaseio.com",
+    "project_id": "status-react-app",
+    "storage_bucket": "status-react-app.appspot.com"
+  },
```

Comment:
This file is the Firebase project json file for configuring the Firebase SDK.
It seems this file is safe to check-in to the codebase, though we should make sure that our Firebase project is configured in a way to avoid misuse of keys.